rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -42,7 +42,7 @@ namespace rsx
 		//Don't throw, gather information and ignore broken/garbage commands
 		//TODO: Investigate why these commands are executed at all. (Heap corruption? Alignment padding?)
 		LOG_ERROR(RSX, "Invalid RSX method 0x%x (arg=0x%x)", _reg << 2, arg);
-		rsx->invalid_command_interrupt_raised = true;
+		//rsx->invalid_command_interrupt_raised = true;
```

Comment:
This should stay. Incorrectly flagged commands should be properly registered, not silently ignored. A lot of times, this is the true sign that everything went wrong and you'll see completely crazy values when corruption happens.
I'm aware of a few methods that are generated by PSLIGHT that we do not have registered.

Reply:
I'm actually not sure about this, in the past we only aborted the command. If its left uncommented we would trigger a full fifo recovery, if the method is valid then we would possibly go into an infinite loop of recovery and unknown method exection.

Reply:
That is why I was saying we need to register any correct methods instead of grouping invalid and possibly valid together. Some homebrew will break temporarily but in the end, we don't miss any resets that should have happened. The way its set up now means only invalid index will trigger fifo reset.